backend/tests/clubs/test_security.py

```diff
@@ -138,7 +138,7 @@ def test_check_detail_level_permissions(self):
         """
 
         # Don't put your function here unless it never returns any private information.
-        whitelist = set()
+        whitelist = {("QuestionAnswerViewSet", "like"), ("QuestionAnswerViewSet", "unlike")}
```

Comment:
Looks like flake8 wants this line to be under 88 characters
```
./tests/clubs/test_security.py:141:89: E501 line too long (92 > 88 characters)
```
can you reformat it? Maybe
```
whitelist = {
    ("QuestionAnswerViewSet", "like"),
    ("QuestionAnswerViewSet", "unlike"),
}
```
would work?